### ISSUE TITLE
[DEMO] Quantum multilabel classification with JAX

### ISSUE THREAD

**Issue by poporubeus** [PennyLaneAI/qml#1096]
#### General information

**Name**
Francesco Aldo Venturelli (poporubeus).

**Affiliation** 
University of Florence.

--------------------------------------------------------------------------------

#### Demo information

**Title**
Quantum multilabel classification with JAX

**Abstract**
Quantum convolutional neural network (QCNN) for multilabel image classification written by combining Pennylane with JAX.
In this simple tutorial we select four different classes of digits images from the sklearn.datasets.load_digits and we construct a quantum convolutional neural network to train and make the classification of multiclass images.
After model training, we select the last updated and optimal parameters (which correspond to the maximum value of the validation accuracy) and use them to test the model on the unseen test images. To have an idea about all the steps needed to complete the experiment we recommend to have a look at the scripts in the /src folder.
Hope this could be useful for you, feel free to use these codes and make further improvements. With this code we would like to emphasize the ability of making multilabel classification and stop of being constrained by binary classifications!

**Relevant links**
https://github.com/poporubeus/quantum_machine_learning/tree/main/multilabel_classification/tutorial

**Comment by poporubeus** [comment]
I submit this simple tutorial on multilabel classification using a QCNN combining Pennylane and JAX together.

**Comment by poporubeus** [comment]
I submit this simple tutorial on multilabel classification using a QCNN combining Pennylane and JAX together.

**Comment by ikurecic** [comment]
Thanks a lot for the submission, @poporubeus !
Could I ask you to add a couple more short comments in the Notebook to explain what you're doing and concluding, in the second half? That would be fantastic.

**Comment by poporubeus** [comment]
I have added more comments on the .ipynb. Let me know if they are sufficient. Thanks in advance.

**Comment by CatalinaAlbornoz** [comment]
Thanks @poporubeus! We'll do another round of review and keep you updated. This may take a few days.

**Comment by ikurecic** [comment]
Thanks a lot for making the updates, @poporubeus , this is great to see — thanks so much for sharing your code!

You'll be able to already see your demo linked on the [PennyLane website](https://pennylane.ai/qml/demos_community/) today or tomorrow, and we will also post it on social media later on.
Congrats! ☺

**Comment by poporubeus** [comment]
Thank you so much. You are a super and dynamic community, and thanks for being so and for giving a constant support.
Thanks 😃,
It's a pleasure.
FAV.

Inviato da Outlook per iOS<https://aka.ms/o0ukef>
________________________________
Da: Ivana Kurečić ***@***.***>
Inviato: Friday, May 10, 2024 1:09:59 PM
A: PennyLaneAI/qml ***@***.***>
Cc: Francesco Aldo Venturelli - ***@***.*** ***@***.***>; Mention ***@***.***>
Oggetto: Re: [PennyLaneAI/qml] [DEMO] Quantum multilabel classification with JAX (Issue #1096)


Thanks a lot for making the updates, @poporubeus<https://github.com/poporubeus> , this is great to see — thanks so much for sharing your code!

You'll be able to already see your demo linked on the PennyLane website<https://pennylane.ai/qml/demos_community/> today or tomorrow, and we will also post it on social media later on.
Congrats! ☺

**Comment by poporubeus** [comment]
Good morning @ivana, that would be awesome!
Thank you so much for this opportunity.
This is my LinedIn account: https://www.linkedin.com/in/francesco-aldo-venturelli-ab5721294/

Thank you,
Kind regards.

FAV.
________________________________
Da: Ivana Kurečić ***@***.***>
Inviato: lunedì 13 maggio 2024 11:36
A: PennyLaneAI/qml ***@***.***>
Cc: Francesco Aldo Venturelli - ***@***.*** ***@***.***>; Mention ***@***.***>
Oggetto: Re: [PennyLaneAI/qml] [DEMO] Quantum multilabel classification with JAX (Issue #1096)


Hi @poporubeus<https://github.com/poporubeus> , thank you so much for your kind words. :)

I just wanted to quickly follow up to ask you if you'd like our Marketing Team to tag your LinkedIn account (or Twitter account) in a social media post. If that would be fine, please go ahead and share the link with me.

Thank you!
